docs/ListTutorial.md

```diff
@@ -43,10 +43,11 @@ const BookList = () => {
     const [filter, setFilter] = useState('');
     const [page, setPage] = useState(1);
     const perPage = 10;
+    const sort = { field: 'id', order: 'ASC' };
```

Comment:
It doesn't make sens to introduce it here IMO. Why `sort` and not `pagination`?

Reply:
Sorry, wrong section